### PR TITLE
Update input configuration.

### PR DESCRIPTION
**Pull request by josiah-tesfu**
- Updated the input configuration of LED. Configuration is initially set through the editor. Once the node is deployed, configuration can be set dynamically from input. If any parameters from the input message are empty, the previously set configurations are used.

- Changed relay state to boolean type.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,16 +15,27 @@ npm install @edgepi-cloud/node-red-edgepi-relay
...
 ### Inputs
-Any message can be used to trigger this node.
+- **payload** *string*<br>
+The relay state.
+
+Example input configuration:
+```
+msg {
+    "payload": "open"
+}
```
Comment (farzadpanahi) [inline README.md]: use boolean

**Review by farzadpanahi — CHANGES_REQUESTED** [review]

File: edgepi-relay.js [hunk 2]
```diff
@@ -1,44 +1,54 @@
...
+    const node = this;
+    let relayState = config.relayState;
+
+    initializeNode(config).then((relay) => {
+      node.on("input", async function (msg, send, done) {
+        node.status({ fill: "green", shape: "dot", text: "input recieved" });
+        try {
+          relayState = msg.payload ?? relayState;
+          const stateStr = relayState === true ? "closeRelay" : "openRelay";
+          msg = {payload: await relay[stateStr]()};
```
Comment (maria-mz) [inline edgepi-relay.js:15]: This is just for my understanding but can you explain `relay[stateStr]()`? Is this part updating the configuration then returning the associated boolean when it's successfully updated using `stateStr`?

**Review by maria-mz — COMMENTED** [review]
looks nice just a clarifying comment

**Review by farzadpanahi — APPROVED** [review]

File: edgepi-relay.js (hunk 2 above)
Comment (josiah-tesfu) [reply edgepi-relay.js:15]: `relay[stateStr]()` is a function call. E.g. if `stateStr` is `openRelay`, this would call `openRelay()`.

**Review by farzadpanahi — CHANGES_REQUESTED** [review]
fix the example payload in readme

**Review by farzadpanahi — APPROVED** [review]

**Pull request merged**